lib/kafo/configuration.rb

```diff
@@ -20,7 +22,7 @@ def self.colors_possible?
         :mapping              => {},
         :answer_file          => './config/answers.yaml',
         :installer_dir        => '.',
-        :modules_dir          => './modules',
+        :module_dirs          => ['./modules'],
```

Comment:
how do we fix existing foreman installations? I wonder if `module_dirs` should default to `[$modules_dir]` somehow or use it if it's not defined